addons/delivery/views/delivery_view.xml

```diff
@@ -275,6 +275,7 @@
                 <data>
                   <xpath expr="//field[@name='payment_term_id']" position="after">
                       <field name="id" invisible="1"/>
+                      <field name="invoice_shipping_on_delivery" string="Extra line on each delivery" attrs="{'readonly': [('state', 'in', ('done'))]}"/>
```

Comment:
Your `string` is imprecise. I would rephrase: "Delivery orders will add shipping costs"

Reply:
Hmm, it is still not precise and clear IMHO if I rephrase like you suggest.

What about: "Each delivery will add extra shipping costs"